src/cargo/ops/cargo_package.rs

```diff
@@ -145,10 +131,6 @@ pub fn package_one(
     dst.file().set_len(0)?;
     let uncompressed_size = tar(ws, pkg, ar_files, dst.file(), &filename)
         .with_context(|| "failed to prepare local package for uploading")?;
-    if opts.verify {
-        dst.seek(SeekFrom::Start(0))?;
-        run_verify(ws, pkg, &dst, opts).with_context(|| "failed to verify package tarball")?
-    }
 
     dst.seek(SeekFrom::Start(0))?;
```

Comment:
I assume the first `seek` is no longer needed because we are relying on the second seek with `run_verify` moved

Reply:
The first `seek` just got moved inside `run_verify`